### ISSUE TITLE
[Bitstamp] WithdrawalRequest object does not support txid field

### ISSUE THREAD

**Issue by tross-nydig** [knowm/XChange#4888]
The Bitstamp V2 API has [an endpoint that gets withdrawal requests](https://www.bitstamp.net/api/#tag/Withdrawals/operation/GetWithdrawalRequests). 

The response object looks like this:

```
{
    "address": "aMDHooGmAkyrsaQiKhAORhSNTmoRzxqWIO",
    "amount": "0.00006000",
    "currency": "BTC",
    "datetime": "2022-01-31 16:07:32",
    "id": 1,
    "network": "bitcoin",
    "status": 2,
    "transaction_id": "NsOeFbQhRnpGzNIThWGBTkQwRJqTNOGPVhYavrVyMfkAyMUmIlUpFIwGTzSvpeOP",
    "txid": 1,
    "type": 0
}
```

The `transaction_id` field is the transaction hash but the response also contains a field called `txid` that represents Bitstamp's internal transaction ID. This field is very useful for anyone who might also call [the endpoint that brings back all Bitstamp user transactions](https://www.bitstamp.net/api/#tag/Transactions-private/operation/GetUserTransactions).

The xchange-bitstamp module doesn't support this field at present but it's a pretty easy thing to add by simply adding `txid` to xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/dto/account/WithdrawalRequest.java

**Comment by rizer1980** [comment]
Hello, [tross-nydig](https://github.com/tross-nydig)
check #4889

**Comment by tross-nydig** [comment]
Awesome, quick work! I had my own pull request in the works but am just happy to have this field in the code now. I might create a small followup PR to add `txId` to the `toString` method if that's OK?

```
        + ", txid="
        + txid
```

**Comment by tross-nydig** [comment]
Added a tiny PR here: https://github.com/knowm/XChange/pull/4896

**Comment by timmolter** [comment]
merged #4896
